datalake/src/main/java/com/sequenceiq/datalake/flow/dr/DatalakeDrSkipOptions.java

```diff
@@ -53,11 +62,12 @@ public boolean equals(Object o) {
             return false;
         }
         DatalakeDrSkipOptions other = (DatalakeDrSkipOptions) o;
-        return skipAtlasMetadata == other.skipAtlasMetadata && skipRangerAudits == other.skipRangerAudits && skipRangerMetadata == other.skipRangerMetadata;
```

Comment:
A question to the code owners - why do we have equals/hashcode here? This object will never be put inside any container. Would it be better to remove them instead?